src/modules/.test.js

```diff
@@ -0,0 +1,156 @@
+//add function
```

Comment:
Great job using Jest library 👍 But there are a few points you need to rectify.

- Please move your test folders into a test folder. It will be more easy to find the tests in your repository.

- .test.js is not descriptive for a test file. Kindly add a proper and descriptive name for your test file.